subsys/net/Kconfig

```diff
@@ -15,18 +15,28 @@ config NET_BUF
 	  This option enables support for generic network protocol
 	  buffers.
 
+if NET_BUF
+
+config NET_BUF_USER_DATA_SIZE
+	int "Size of user_data available in every network buffer"
+	default 8
+	range 8 65535
```

Comment:
Why it can't be 4? Why can't be 1 (just some flag)? Why default is 8?

Reply:
You're right. This could even be 0. However then we either need Kconfig dependencies or build asserts to make sure the networking and Bluetooth stacks don't get built with anything smaller than what they need (e.g. Bluetooth needs 8)